### ISSUE TITLE
Three questions

### ISSUE THREAD

**Issue by robert-hoyt** [pycaret/pycaret#239]
1. I emailed Moez about a month ago asking if you planned to add a neural network to your list of algorithms. I'll be talking to some AI-oriented physicians about PyCaret and they may ask

2. When I code "plot_model(lr)" I expected multiple options like your guide's show: AUC, confusion matrix, etc. All I get is the AUC plot. When I use "evaluate_model(lr)" it gives some numbers but not the multiple options your user guide shows.

3. I want to confirm your validation process. The data is divided into 70% training, 30% testing. You use k-10 fold cross-validation to train and compare models. True testing is done with the 30% data when you "predict_model(lr)". Thanks

**Comment by pycaret** [comment]
@robert-hoyt 

Hi Robert, Sorry for delay in responding. 

See the answers to your question:

1. At the point the only NN implementation in pycaret.classification and pycaret.regression is MLP. We donot plan to extend the package to deep learning at this time.

2. Which environment are you using for evaluate_model(). Since it uses html widgets you can only use that function in Notebook only. If you want to generate a specific plot you can do that by passing plot parameter within plot_model function. For example to get confusion matrix you can run this plot_model(model, plot='confusion_matrix'). To see complete list of available plots, please check https://www.pycaret.org/plot-model 

3. Your understanding is correct. K-fold CV is done on training set only (70% by default) and when you use predict_model() the table prints metrics on hold-out set (30% by default).

Thanks.

**Comment by robert-hoyt** [comment]
1. I see on the PyCaret website that MLP is available but it has never appeared as an algorithm on multiple classifications I have run? reason. That is why I was unaware that PyCaret included MLP or ANN. My last classification model in Jupyter Lab used 14 algorithms, but not MLP

2. I know I can code multiple plot models one at a time but wanted that automatically generated. I am currently using Jupyter Lab but used Jupyter Notebooks to code Python and PyCaret in the past. Is that the issue?

**Comment by robert-hoyt** [comment]
I did notice that my results were the same for Jupyter Notebook as it was for Jupyter Lab with classification models run on the heart disease prediction dataset. Because PyCaret ranks model performance based on descending accuracy, is it possible that performance by MLP is not included because it's performance is below a certain threshold?

**Comment by pycaret** [comment]
@robert-hoyt Hey Robert,

The reason performance of MLP is not included is because by default turbo mode is set to True in compare_models which prevents training algorithms which may take longer times. MLP is one of them. That said, it doesn't mean its not available. You can still train MLP by passing it in create model (example: create_model('mlp') ).

If you would like to include it in compare_models, just set Turbo parameter to False inside of compre_models. 

Hope this helps. Let me in know how it goes.

**Comment by robert-hoyt** [comment]
Thanks. I did create a MLP model but it produced an AUC mean of .325, compared to a mean logistic regression AUC of .87
The other surprise was a test AUC for logistic regression AUC of .95 suggesting underfitting (I think)
